.github/workflows/on-push.yml

```diff
@@ -205,13 +205,13 @@ jobs:
           coverage combine **/.coverage
           coverage report
           coverage xml
-      - name: Coveralls
+      - name: Coveralls-Action
         uses: coverallsapp/github-action@v2
-      - name: Coveralls v2
+      - name: Coveralls-Comment
```

Comment:
```suggestion
      - name: Coveralls-Comment
         env:
             COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
            
```

Reply:
and then you don't need to change the script at all

Reply:
good idea, done